tslint.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": ["tslint:recommended"],
+    "rules": {
+        "no-var-requires": false,
+        "ordered-imports": false,
+        "quotemark": [true, "single", "avoid-template", "avoid-escape"],
```

Comment:
@ashleydavis just need to change this from `"single"` to `"double"` and run the `npm run fix-lint` to change to double quotes across `src`

Reply:
I changed this to `"double"` already and changed the single quotes to double quotes